frontend/src/metabase/admin/upsells/components/Upsells.base.module.css

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
I'm extracting vars to a separate CSS module, so they can be imported by other modules.